Tests/test_GenomeDiagram.py

```diff
@@ -1070,6 +1073,25 @@ def test_diagram_via_object_pdf(self):
         gdd.write(output_filename, 'PDF')
 
 
+def _check_prereqs():
+    """Check that the input data is available"""
+    if not os.path.exists("Graphics"):
+        os.mkdir("Graphics")
+    if not os.path.exists("GenBank"):
+        os.mkdir("GenBank")
+
+    gbfilename = os.path.join("GenBank", "NC_005816.gb")
+    if not os.path.exists(gbfilename):
```

Comment:
Unless the test script is being run from the wrong directory, this will exist - https://github.com/biopython/biopython/blob/master/Tests/GenBank/NC_005816.gb

(I may be missing something, but at face value I would revert this commit)

Reply:
Ah, I've apparently been running tests the wrong way. Reverted.